rcsb/workflow/targets/ProteinTargetSequenceWorkflow.py

```diff
@@ -128,6 +129,8 @@ def exportTargetsFasta(self, resourceNameList=None, useCache=True, addTaxonomy=F
             addTaxonomy (bool, optional): add taxonomy details to each target record. Defaults to False.
             reloadPharos (bool, optional): reload Pharos target resources from SQL dump. Defaults to False.
             fromDbPharos (bool, optional): export Pharos target resources from local database server. Defaults to False.
+            backup (bool, optional): export Pharos target resources from local database server. Defaults to False.
```

Comment:
Maybe we should name this `backupPharos`, since this method also handles other resources? Also would keep them in line with the other pharos-specific options.

Reply:
I like that idea, thanks.